src/zoid/buttons/util.js

```diff
@@ -357,3 +362,60 @@ export function getButtonSize(
     }
   }
 }
+
+export const getModal: (
+  clientID: string,
+  merchantID: $ReadOnlyArray<string> | void
+) => Object = memoize(async (clientID, merchantID) => {
+  try {
+    const namespace = getNamespace();
+    if (!window[namespace]?.MessagesModal) {
```

Comment:
I think we can be confident that `window[namespace]` will exist otherwise the SDK would error out while initializing. Are we able to remove the optional chaining to reduce doubt of its existence?